src/pages/CartPage.ts

```diff
@@ -41,4 +41,16 @@ export class CartPage extends AbstractPage {
       await expect(this.emptyCartMessage).toBeVisible();
     });
   }
+
+  async assertCoffeItemPresenceInCart(productName: string): Promise<void> {
```

Comment:
Try to avoid duplicates, how this can be done:

```
  async removeButtonLocator(productName: string): Promise<Locator> {
    return this.page.getByRole('button', { name: `Remove one ${productName}`, exact: true });
  }

  async assertCoffeItemPresenceInCart(productName: string): Promise<void> {
    await test.step(`Checks if "${productName}" is present in the cart`, async() => {
      await expect(await this.removeButtonLocator(productName)).toBeVisible();
    });
  }

  async assertCoffeItemAbscenceInCart(productName: string): Promise<void> {
    await test.step(`Checks if "${productName}" is NOT present in the cart`, async() => {
      await expect(await this.removeButtonLocator(productName)).toBeHidden();
    });
  }
```

Reply:
fixed